charts/argo-cd/values.yaml

```diff
@@ -266,6 +285,8 @@ configs:
     server.disable.auth: false
     # -- Enable GZIP compression
     server.enable.gzip: true
+    # -- Enable proxy extension feature
```

Comment:
Can we mention the alpha phase here, please?

```suggestion
    # -- Enable proxy extension feature. (proxy extension is in Alpha phase)
```

Reply:
@mkilchhofer 

Thank you for your feedback. The changes have been committed. Please kindly review.

Reply:
@charlychiu thank you for implementing the propoal 👍